src/walk_dirs.rs

```diff
@@ -1,13 +1,12 @@
-extern crate find_ts_imports;
-extern crate serde;
 use crate::fence::{parse_fence_file, Fence};
-use find_ts_imports::{parse_source_file_imports, SourceFileImportData};
+use crate::file_import_retriever;
 use jwalk::WalkDirGeneric;
 use relative_path::RelativePath;
 use serde::Deserialize;
-use std::collections::HashSet;
+use std::collections::{HashSet, HashMap};
 use std::iter::FromIterator;
 use std::path::{Path, PathBuf};
+use file_import_retriever::{get_imports_from_file, get_imports_map};
```

Comment:
This could be condensed with the import on like 2 to `crate::file_import_retriever::{...}`.

Reply:
Though to be honest the imports are a mess to start with, apologies for that.